aten/src/ATen/native/cpu/avx_mathfun.h

```diff
@@ -1,4 +1,4 @@
-#ifdef USE_AVX2
+#pragma once
```

Comment:
We might want to avoid changing this file, because it's really a header only library

Reply:
We've also added "inline" to every function as otherwise we get linker conflicts.

Reply:
Header guards (or `#pragma once`) are important for header only libraries too